### ISSUE TITLE
Source highlighting - Only first module in "with" turns into a link

### ISSUE THREAD

**Issue by Grinnz** [CPAN-API/metacpan-web#1614]
See https://metacpan.org/source/DBOOK/Dist-Zilla-PluginBundle-Author-DBOOK-0.020/lib/Dist/Zilla/PluginBundle/Author/DBOOK.pm#L5

**Comment by Grinnz** [comment]
Also, https://metacpan.org/source/ETHER/Dist-Zilla-PluginBundle-Author-ETHER-0.107/lib/Dist/Zilla/PluginBundle/Author/ETHER.pm#L11 for some more examples that might be trickier to handle...

**Comment by haarg** [comment]
I don't think it will be reasonable to try to fix this without replacing the syntax highlighter.  I have an in progress branch to do that which I believe already solves this (multiple, and split lines), but I haven't touched it in a while, and it doesn't yet implement all of the things we've hacked onto the current syntax highlighter.
